src/Compilers/Core/Desktop/AnalyzerFileReference.cs

```diff
@@ -208,13 +213,12 @@ internal void AddAnalyzers(ImmutableArray<DiagnosticAnalyzer>.Builder builder, s
                 // If there are no analyzers, don't load the assembly at all.
                 if (!analyzerTypeNameMap.ContainsKey(language))
                 {
-                    this.AnalyzerLoadFailed?.Invoke(this, new AnalyzerLoadFailureEventArgs(AnalyzerLoadFailureEventArgs.FailureErrorCode.NoAnalyzers, null, null));
```

Comment:
Turns out we already have a test that covers this situation:
https://github.com/dotnet/roslyn/blob/0c84e8f8fccd7332c54a6c40c27240af7574ae7e/src/Compilers/Core/CodeAnalysisTest/AnalyzerFileReferenceTests.cs#L201